Weather/Weather/Utils/Constants.swift

```diff
@@ -24,4 +24,7 @@ struct Constants {
     static let invalidRequest = "INVALID_REQUEST"
     static let blank = ""
     static let warning = "Warning"
+    static let widthScreen = UIScreen.main.bounds.width
+    static let heightScreen = UIScreen.main.bounds.height
+    static let nameStoryboard = "Main"
```

Comment:
Tổng hợp Storyboard ra một file storyboards.swift riêng nhé
```suggestion
    struct Storyboard {
           static let main = UIStoryboard(name: "Main", bundle: nil)
    }
```